gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -389,6 +382,22 @@ private int readBufferedContentInto(ByteBuffer byteBuffer) {
     return bytesToWrite;
   }
 
+  private boolean canReadFromExistingRequest(ByteBuffer byteBuffer) {
```

Comment:
can we have the private methods after the public methods? Will help with readability.